src/main/java/DatabaseInteractin/ExecuteQuerys.java

```diff
@@ -0,0 +1,42 @@
+package DatabaseInteractin;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Scanner;
+
+public class ExecuteQuerys {
+  DataBaseInteracting dataBaseInteracting = new DataBaseInteracting();
+  Scanner sc = new Scanner(System.in);
```

Comment:
Not using this?
Actually server code should not have to interact with `System.in`